app/components/CrashLocationBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import Alert from "react-bootstrap/Alert";
+
+interface CrashLocationBannerProps {
+  privateDriveFlag: boolean | null;
+}
+
+/**
+ * Crash Location Banner is rendered if private flag drive is true or austin full purpose flag is false.
+ */
+export default function CrashLocationBanner({
+  privateDriveFlag,
+}: CrashLocationBannerProps) {
+  return (
+    <Alert variant="warning">
+      This crash is not included in Vision Zero statistical reporting because{" "}
+      {privateDriveFlag
+        ? "it occurred on a private drive"
+        : "it is located outside of the Austin full purpose jurisdiction"}
```

Comment:
Do you mind period at the ends of this sentence?

Reply:
I took the copy directly from the existing banner https://github.com/cityofaustin/vision-zero/blob/main/editor/src/views/Crashes/Crash.js#L151 and I wasn't sure if the lack of `.` was intentional. I can add it!

Reply:
It was! And I ran off to find some blurbage that aligns with my arbitrary opinion:
https://ux.stackexchange.com/questions/122147/periods-in-ux-ui-microcopy

I actually I think this is a convention that Amenity introduced and may even be written in a style guide of ours somewhere...

Reply:
That's cool - I'm totally good if that's a style convention we want to adopt, but given a vote, I'd still be of the mind that a cogent collection of words that includes a subject and predicate deserves a period, but I do get it - and I think I'm in the minority -- I am super cool either way!!

Reply:
Thanks, Frank—I definitely appreciate your attention to the details. It would be kinda nice to put these decisions into a VZ style guide that might live in the repo. Maybe something that could be done as part of https://github.com/cityofaustin/atd-data-tech/issues/20013.